### PR TITLE
integration with with cookiecutter

### PR DESCRIPTION
**Pull request by jabelk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,114 +3,54 @@ This repository is a template to be used when creating a new repository containi
...
-If the repo includes automated tests, detail how to run those tests.
-If the repo is instrumented with a continuous testing framework, that is even better.
+You have two options, if you have familiarity with Python, use the cookiecutter steps. Otherwise you can manually create a copy of this repo template. 
 
 
-## Known issues
+### Cookiecutter Automated Repo Creation
 
-Document any significant shortcomings with the code. If using [GitHub Issues](https://help.github.com/en/articles/about-issues) to track issues, make that known and provide any templates or conventions to be followed when opening a new issue. 
+This process usies [cookiecutter](https://github.com/audreyr/cookiecutter) to auto-generate the files for you. This is helpful if you create multiple use cases.
```
Comment (eckelcu) [inline README.md]: typo s/usies/uses

**Review by eckelcu — CHANGES_REQUESTED** [review]
Looks great. I did not test the various paths and links under the new structure, did you?
I noticed one typo while reviewing - see comment inline.

**Comment by jabelk** [comment]
Pathing appears to work, will fix typo and commit again, sample output:
```bash
pull$ git clone https://github.com/CiscoDevNet/code-exchange-repo-template.git
Cloning into 'code-exchange-repo-template'...
remote: Enumerating objects: 11, done.
remote: Counting objects: 100% (11/11), done.
remote: Compressing objects: 100% (9/9), done.
remote: Total 99 (delta 5), reused 6 (delta 2), pack-reused 88
Unpacking objects: 100% (99/99), done.
pull$ cd code-exchange-repo-template/
code-exchange-repo-template$ git pr 4
remote: Enumerating objects: 13, done.
remote: Counting objects: 100% (13/13), done.
remote: Compressing objects: 100% (9/9), done.
remote: Total 11 (delta 1), reused 11 (delta 1), pack-reused 0
Unpacking objects: 100% (11/11), done.
From https://github.com/CiscoDevNet/code-exchange-repo-template
 * [new ref]         refs/pull/4/head -> pr/4
Switched to branch 'pr/4'
code-exchange-repo-template$ code .
code-exchange-repo-template$ cd ..
pull$ cookiecutter code-exchange-repo-template
project_name [my-awesome-devnet-code-exchange-project]: it-works
project_description [baseline DevNet Code Exchange Project]: my-desc
author_name [Your Name Here]: Lorem Ipsum
author_email [youremail@domain.com]: lorem@domain.com
pull$ tree it-works/
it-works/
├── LICENSE
├── NOTICE
└── README.md

0 directories, 3 files
pull$ ls
code-exchange-repo-template	it-works
pull$ cat it-works/
.gitignore  LICENSE     NOTICE      README.md
pull$ cat it-works/LICENSE
BSD 3-Clause License

Copyright (c) 2019, Cisco Systems, Inc. and/or its affiliates

Redistribution and use in source and binary forms, with or without
modification, are permitted provided that the following conditions are met:

* Redistributions of source code must retain the above copyright notice, this
  list of conditions and the following disclaimer.

* Redistributions in binary form must reproduce the above copyright notice,
  this list of conditions and the following disclaimer in the documentation
  and/or other materials provided with the distribution.

* Neither the name of the copyright holder nor the names of its
  contributors may be used to endorse or promote products derived from
  this software without specific prior written permission.

THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
pull$ cat it-works/
.gitignore  LICENSE     NOTICE      README.md
pull$ cat it-works/NOTICE
it-works

Copyright (c) 2019, Cisco Systems, Inc. and/or its affiliates

This project includes software developed at Cisco Systems, Inc. and/or its affiliates.
pull$ cat it-works/README.md
# it-works

my-desc

![add-image-here]()

## Use Case Description

Describe the problem this code addresses, how your code solves the problem, challenges you had to overcome as part of the solution, and optional ideas you have in mind that could further extend your solution.

## Installation

Detailed instructions on how to install, configure, and get the project running. Call out any dependencies. This should be frequently tested and updated to make sure it works reliably, accounts for updated versions of dependencies, etc.

## Configuration

If the code is configurable, describe it in detail, either here or in other documentation that you reference.

## Usage

Show users how to use the code. Be specific.
Use appropriate formatting when showing code snippets or command line output.

### DevNet Sandbox

A great way to make your repo easy for others to use is to provide a link to a [DevNet Sandbox](https://developer.cisco.com/site/sandbox/) that provides a network or other resources required to use this code. In addition to identifying an appropriate sandbox, be sure to provide instructions and any configuration necessary to run your code with the sandbox.

## How to test the software

Provide details on steps to test, versions of components/dependencies against which code was tested, date the code was last tested, etc.
If the repo includes automated tests, detail how to run those tests.
If the repo is instrumented with a continuous testing framework, that is even better.


## Known issues

Document any significant shortcomings with the code. If using [GitHub Issues](https://help.github.com/en/articles/about-issues) to track issues, make that known and provide any templates or conventions to be followed when opening a new issue.

## Getting help

Instruct users how to get help with this code; this might include links to an issues list, wiki, mailing list, etc.

**Example**

If you have questions, concerns, bug reports, etc., please create an issue against this repository.

## Getting involved

This section should detail why people should get involved and describe key areas you are currently focusing on; e.g., trying to get feedback on features, fixing certain bugs, building important pieces, etc. Include information on how to setup a development environment if different from general installation instructions.

General instructions on _how_ to contribute should be stated with a link to [CONTRIBUTING](./CONTRIBUTING.md) file.

## Author(s)

This project was written and is maintained by the following individuals:

* Lorem Ipsum <lorem@domain.com>
pull$
```

File: README.md (hunk 1 above)
Comment (jabelk) [reply README.md]: @eckelcu , thanks! Typo addressed.

**Review by eckelcu — APPROVED** [review]
Thanks!

**Pull request merged**